lib/cocoapods/target/build_settings.rb

```diff
@@ -1246,6 +1292,17 @@ def user_target_xcconfig_values_by_consumer_by_key
           merged_xcconfigs(user_target_xcconfig_values_by_consumer_by_key, :user_target_xcconfig)
         end
 
+        # @see BuildSettings#should_apply_xctunwrap_fix?
+        def should_apply_xctunwrap_fix?
+          target.library? && target.user_targets.any? do |ut|
+            next false unless Version.new(ut.deployment_target) < Version.new('12.2')
```

Comment:
should this only be for iOS?

Reply:
I think so. I can limit it to that but I will check if it works or not on macOS first.

Reply:
Seems to work without the need for the fix on macOS. Will scope it on iOS only.